App.global.scss

```diff
@@ -11,68 +11,6 @@ body {
   margin: 0;
 }
 
-/* header attrs when scrolled < X pixels (set in Header component) from top */
```

Comment:
Remove all of these in this global css file

The Global file should only be styles that are used anywhere/everywhere

The component specific ones should be things like `header-top` where they are specifically for one component or element